primitives/externalities/src/lib.rs

```diff
@@ -36,6 +36,9 @@ mod scope_limited;
 ///
 /// Provides access to the storage and to other registered extensions.
 pub trait Externalities: ExtensionStore {
+	/// Write a key value pair to the offchain storage database.
+	fn local_ocw_storage_write_kv(&mut self, key: &[u8], value: &[u8]);
```

Comment:
I think this should rather be called `fn set_offchain_storage(&mut self, key: &[u8], value: Option<&[u8]>)`